docs/installation.md

```diff
@@ -33,17 +33,94 @@
 
 ---
 
-You just created two binary files: vatz and vatz-plugin to run.  
+# Usage
+## 1. Modify default.yaml (VATZ)
+There's yaml file as config, and default config must be updated by yourself to set up VATZ service and clients
+sample:
+> This is default yaml, please update secrets, etcs for your node use.
+```
+vatz_protocol_info:
+  protocol_identifier: "Put Your Protocol here"
+  port: 9090
+  health_checker_schedule:
+    - "0 1 * * *"
+  notification_info:
+    host_name: "Put your machine's host name"
+    default_reminder_schedule:
+      - "*/15 * * * *"
+    dispatch_channels:
+      - channel: "discord"
+        secret: "Put your Discord Webhook"
+      - channel: "pagerduty"
+        secret: "Put your PagerDuty's Integration Key (Events API v2)"
+      - channel: "telegram"
+        secret: "Put Your Bot's Token"
+        chat_id: "Put Your Chat's chat_id"
+        reminder_schedule:
+          - "*/5 * * * *"
+  rpc_info:
+    enabled: true
+    address: "127.0.0.1"
+    grpc_port: 19090
+    http_port: 19091
+    
+plugins_infos:
+  default_verify_interval: 15
+  default_execute_interval: 30
+  default_plugin_name: "vatz-plugin"
+  plugins:
+    - plugin_name: "samplePlugin1"
+      plugin_address: "localhost"
+      plugin_port: 9001
+      verify_interval: 7
+      execute_interval: 9
+      executable_methods:
+        - method_name: "sampleMethod1"                                                                                     
+```
+### VATZ protocol Infos
+1. `port`: check your machine used port number. If you set it to a port number that is already in use, an error will occur.
+2. `health_checker_schedule`: use cron expression to set time to check if vatz is alived. 
+3. `dispatch_channels`: add infos to recieve arlerts through discord, pagerduty, and telegram.
+4. `reminder_schedule`: use cron expression to set time to resend the alert if it is not confirmed.
 
-## Usage
-- Modify default.yaml (VATZ)
-    - plugin_name
-        - refer to `vatz-plugin-common/plugins/cosmos-sdk-blocksync/main.go` of official repository
-            - Search `pluginName`in vat-plugin  and update the default.yaml with `pluginName`
-    - Port
-        - Check your machine used port number. If you set it to a port number that is already in use, an error will occur.
-    - discord_secret
-        - If you want to receive discord messages, register webhook URL.
 
-- Execute each binary as below <br> <img width="1510" alt="image" src="https://user-images.githubusercontent.com/106724973/180962626-c4de859c-526d-4038-87b8-badebed44136.png">
-- Alert notification will be sent if block height is not increased <br> <img width="223" alt="image" src="https://user-images.githubusercontent.com/106724973/180962941-4db55b66-ceb6-4cd4-bd53-96d21bdf5596.png">
+### Plugin Infos
+> **You can find further plugins here.**
+> - [vatz-plugin-common](https://github.com/dsrvlabs/vatz-plugin-common)
```

Comment:
@qkrybin @rootwarp 
I don't think `vatz-plugin-common` is our official plugin, do we need to put this on here?
what do you guys think? 

I hope not to put here unless we keep manage the plugin.

Reply:
@xellos00 Oh, you're right.
For this document, I think only including the official plugins will be good.

Reply:
@qkrybin 
Can you get rid of vatz-plugin-common from installation.md?

Reply:
I removed it!